pkg/client/unversioned/request_test.go

```diff
@@ -729,6 +729,7 @@ func TestDoRequestNewWay(t *testing.T) {
 	defer testServer.Close()
 	c := NewOrDie(&Config{Host: testServer.URL, Version: testapi.Default.Version(), Username: "user", Password: "pass"})
 	obj, err := c.Verb("POST").
+		GroupVersion(testapi.Default.Version()).
```

Comment:
Why not use testapi.Default.GroupAndVersion()? Although they both return "v1", using GroupAndVersion() looks more consistent.